lib/src/sdk.ts

```diff
@@ -75,6 +96,29 @@ export class ExchangeSDK {
   private transport: WindowMessageTransport | undefined;
   private logger: Logger = new Logger();
 
+  private transactionStrategy: {
+    [K in Transaction["family"]]: TransactionStrategyFunction;
+  } = {
+    bitcoin: withoutGasLimitTransaction,
```

Comment:
Oh I think we want to use defaultTransaction here, it was a mistake on previous switch case. 
Your refactor cleaned this error, thanks :)
```suggestion
    bitcoin: defaultTransaction,
```